backend/inspirehep/submissions/views.py

```diff
@@ -54,13 +54,25 @@
 LOGGER = structlog.getLogger()
 
 
-def get_updated_record_data(record, update_form_data, optional_field_names):
+def get_updated_record_data(
+    record, update_form_data, optional_field_names, fields_to_extend=[]
+):
     record_data = dict(record)
 
     # delete optional fields if they are removed in the form data
     for field in optional_field_names:
         if field not in update_form_data and field in record_data:
             del record_data[field]
+    fields_to_extend = {
+        field: value
+        for (field, value) in update_form_data.items()
+        if field in fields_to_extend
+    }
+    extended_fields = {
+        field: value + record[field] for field, value in fields_to_extend.items()
+    }
```

Comment:
unfortunately, this will cause duplicates.

Reply:
Yes, but in `_private_notes` do we want to replace the note `{"value": "a value"}` or do we want to append it to the list of `_private_notes`?